index.html

```diff
@@ -4,15 +4,246 @@
+        <nav class="nav-top">
+            <a href="#" class="header-logo">Welcome</a>
+            <button class="hamburger" id="menu-bar" type="button">
+               <i class="fa fa-solid fa-bars"></i>
+            </button>
+            <ul class="header-nav-desktop">
+                    <li><a href="">Portifolio</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href="">Contact</a></li>
+            </ul>
+        </nav>
+        <div class="mobile-menu" id="menu-mobile">
+            <div class="menu-header">
+                <span class="close">&times;</span>
+            </div>
+            <div class="menu-body">
+                <ul>
+                    <li><a href="#work-id">Portfolio</a></li>
+                    <li><a href="#about-id">About</a></li>
+                    <li><a href="#contact-id">Contact</a></li>
+                </ul>
+            </div>
+        </div>
 
+    <!--Headline starts Here-->
+    <div class="headline">
+        <div class="header-title">
+            <h1>Hey there. I’m Brhanu. </h1>
+            <h2><span class="h-span">I’m a software developer</span></h2>
+        </div>
+        <div class="heder-info">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-link">
+            <ul class="medias">
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa fa-github"></i></a></li>
+                <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+            </ul>
+        </div>
+        <div class="head-btn">
+            <button type="button">
+                <i class="fa fa-solid fa-angle-down"></i>
+            </button>
+        </div>
+    </div>
+</header>
+<!--Header Ends Here-->
+<!--Work card content starts Here-->
+    <section class="work-card" id="work-id">
+        <h1 class="myrecentwork">My Recent Works</h1>
+        <img class="ind" src="images/Indicator.png" alt="Indicator image goes here">
+        <div class="work-container-card">
+            <div class="work-container">
+                <div class="img-placer">
+                    <img src="images/Axum1.png" alt="Multi-Post Stories Gain+Glory">
+                </div>
+                <div class="txt-info">
+                    <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                    <ul class="languages">
+                        <li><a href="">Ruby on rails</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">JavaScript</a></li>
+                        <li><a href="">html</a></li>
```

Comment:
- [ ] [ OPTIONAL ] For better user experience, these list items (technologies used) should be plain texts and not links since they are only supposed to show a list of the technologies you used without redirect the user. You can consider replacing the `a` tag with `p` tag.